discord/embeds.py

```diff
@@ -615,6 +647,19 @@ def fields(self) -> List[_EmbedFieldProxy]:
         """
         return [EmbedProxy(d) for d in getattr(self, "_fields", [])]  # type: ignore
 
+    @fields.setter
+    def fields(self, value: List[EmbedField]) -> None:
```

Comment:
I think this should be named something different than `fields` since that's the name of the property
```suggestion
    def fields_setter(self, value: List[EmbedField]) -> None:
```

Reply:
Python required property setters to be named the same as the property iirc

```
>>> class f:
...     def __init__(self, a):
...             self.a = a
...     @property
...     def b(self):
...             return self.a
...     @b.setter
...     def b_setter(self, thing):
...             self.a = thing
...
>>> a = f(1)
>>> a.b
1
>>> a.b = 2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: can't set attribute 'b'
>>>
```

Reply:
@plun1331 is correct re: setter naming. It also follows the same naming convention as every other property on the object that has a setter defined.

Reply:
I see